modules/apps/frontend-js/frontend-js-web/test/liferay/util/format_storage.es.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+import formatStorage from '../../../src/main/resources/META-INF/resources/liferay/util/format_storage.es';
+
+describe('Liferay.Util.formatStorage', () => {
```

Comment:
We are missing tests for incorrect parameter types, see https://github.com/liferay/liferay-portal/blob/c4305106ea60aafe4283eb18954a0fe7a141a328/modules/apps/frontend-js/frontend-js-web/test/liferay/util/format_xml.es.js#L7